day_two_guide/topics/proc_backing-up-etcd.adoc

```diff
@@ -60,55 +61,73 @@ Before backing up etcd:
 * Ensure that the {product-title} API service is running.
 * Ensure connectivity with the etcd cluster (port 2379/tcp).
 * Ensure the proper certificates to connect to the etcd cluster.
-* Ensure `go` is installed.
 
 ifeval::["{context}" == "environment-backup"]
-. To ensure the etcd cluster is working, check its health.
-** Run the following command:
+* Ensure that the etcd cluster is working by checking its health:
+** Check the health of the endpoints:
++
+// TODO: is this okay to update to use etcdctl3 (since etcdctl doesn't exist on master node)? Remove ETCDCTL_API=3?
 +
 ----
-# ETCDCTL_API=3 etcdctl --cert="/etc/etcd/peer.crt" \
+# ETCDCTL_API=3 etcdctl3 --cert="/etc/etcd/peer.crt" \
```

Comment:
@geliu2016 @hexfusion Is this change to use `etcdctl3` okay, since it already exists on the host?

Or does it need to stay as `etcdctl`, and then if so, we probably need to add the prerequisite steps on insalling `etcdctl` like are here: http://file.rdu.redhat.com/~ahoffer/2020/BZ-1726771/admin_guide/assembly_restoring-cluster.html#restoring-etcd-data

Reply:
issue here is if they want to backup to a location that is not the mount point. but as we document that using etcdctl3 should be fine. but we can remove the cert flags as they are prepopulated.

Reply:
I believe the command could simply be
```
etcdctl3  \
  --endpoints="https://master-0.example.com:2379,https://master-1.example.com:2379,https://master-2.example.com:2379" \
  endpoint health

Reply:
@hexfusion I updated to remove `ETCDCTL_API=3`, but are you saying that you also don't need to pass in `--cert` and `--key` and `--cacert`?